### PR TITLE
Centrifuge App: Loan portfolio runtime api change

### PR DESCRIPTION
**Pull request by onnovisser**
### Description

This pull request...

Closes #<issue-number>

### Approvals

- [ ] Dev

### CODE REVIEW

File: centrifuge-js/src/CentrifugeBase.ts [hunk 1]
```diff
@@ -113,8 +113,115 @@ const parachainTypes = {
...
+            },
+            notional: 'u128',
+          },
+          outstanding_quantity: 'u128',
+          interest: 'ActiveInterestRate',
+        },
+      },
+    },
+    total_borrowed: 'u128',
+    total_repaid: 'u128',
```
Comment (lemunozm) [inline centrifuge-js/src/CentrifugeBase.ts:218]: This has changed and should be a `RepaidAmount`

File: centrifuge-js/src/CentrifugeBase.ts [hunk 2]
```diff
@@ -113,8 +113,115 @@ const parachainTypes = {
...
+        External: {
+          info: {
+            price_id: {
+              _enum: {
+                Isin: '[u8;12]',
+              },
+            },
+            max_borrow_amount: {
+              _enum: {
+                NoLimit: 'NoLimit',
```
Comment (lemunozm) [inline centrifuge-js/src/CentrifugeBase.ts:206]: This should be `null` I guess

File: centrifuge-js/src/modules/pools.ts [hunk 3]
```diff
@@ -2182,14 +2183,16 @@ export function getPoolsModule(inst: Centrifuge) {
...
+              outstandingPrincipal: CurrencyBalance
+              outstandingInterest: CurrencyBalance
               presentValue: CurrencyBalance
             }
           > = {}
           ;(rawPortfolio as any).forEach(([key, value]: [Codec, Codec]) => {
             const data = value.toPrimitive() as any
             activeLoansPortfolio[String(key.toPrimitive())] = {
-              interestAccrued: new CurrencyBalance(data.interest_accrued, 27), // not sure
+              outstandingPrincipal: new CurrencyBalance(data.outstanding_principal, 27), // not sure about the decimals
```
Comment (lemunozm) [inline centrifuge-js/src/modules/pools.ts:2194]: They are the same as the currency pool it is related

**Review by offerijns — APPROVED** [review]
🚀

**Pull request merged**